gnomad_qc/v5/sample_qc/relatedness.py

```diff
@@ -0,0 +1,180 @@
+"""Script to compute relatedness estimates among pairs of samples in the joint gnomAD v4 (exomes + genomes) + AoU callset."""
+
+import argparse
+import logging
+import textwrap
+
+import hail as hl
+
+# from gnomad_qc.resource_utils import check_resource_existence
+from gnomad_qc.v5.resources.basics import get_logging_path
+from gnomad_qc.v5.resources.constants import WORKSPACE_BUCKET
+from gnomad_qc.v5.resources.sample_qc import (
+    get_cuking_input_path,
+    get_cuking_output_path,
+    get_joint_qc,
+)
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("relatedness")
+logger.setLevel(logging.INFO)
+
+
+def print_cuking_command(
+    cuking_input_path: str,
+    cuking_output_path: str,
+    min_emission_kinship: float = 0.5,
+    cuking_split_factor: int = 4,
+    location: str = "us-central1",
+    machine_type: str = "a2-highgpu-1g",
+    accelerator_count: int = 1,
+    accelerator_type: str = "nvidia-tesla-a100",
+    requester_pays_project: str = "terra-vpc-sc-93ccd8d2",
+) -> None:
+    """
+    Print the command to submit a dsub job for running cuKING.
+
+    :param cuking_input_path: Path to the cuKING input Parquet files.
+    :param cuking_output_path: Path to the cuKING output Parquet files.
+    :param min_emission_kinship: Minimum kinship threshold for emitting a pair of
+        samples in the relatedness output.
+    :param cuking_split_factor: Split factor to use for splitting the full relatedness
+        matrix table into equally sized submatrices that are computed independently to
+        parallelize the relatedness computation and decrease the memory requirements.
+        For example, to halve memory requirements, the full matrix can be split into
+        equally sized submatrices (i.e. a 'split factor' of 4). Only the
+        'upper triangular' submatrices need to be evaluated due to the symmetry of the
+        relatedness matrix, leading to 10 shards. Default is 4.
+    :param location: Location to run the dsub job. Default is "us-central1".
+    :param machine_type: Machine type to use for the dsub job. Default is "a2-highgpu-1g".
+    :param accelerator_count: Number of accelerators to use for the dsub job. Default is 1.
+    :param accelerator_type: Type of accelerator to use for the dsub job.
+        Default is "nvidia-tesla-a100".
+    :param requester_pays_project: Project ID to use for requester pays buckets.
+        Default is "terra-vpc-sc-93ccd8d2".
+    :return:
+    """
+    logger.info(
+        "Printing a command that can be used to submit a dsub job to run "
+        "cuKING on the files created by --prepare-cuking-inputs."
+    )
+    logger.warning(
+        "This printed command assumes that the cuKING directory is in the same "
+        "location where the command is being run and that $PROJECT_ID is set!"
+    )
+    SPLIT_FACTOR = cuking_split_factor
+    TOTAL_SHARDS = SPLIT_FACTOR * (SPLIT_FACTOR + 1) / 2
```

Comment:
Maybe just add a comment for what TOTAL_SHARDS is and why it's computed this way. 

Also, since these are not constants, do they need to be capitalized and does split factor need to be redefined?

Reply:
Ah, so I think that is all run on the terminal, so they should be included in the print

Reply:
I don't know enough about how the quotes are handled to know if the dsub --command will work as expected, but it looks the same as in the screenshot, so if that worked, then it should be fine